tests/e2e/specs/shopper/filter-products-by-price.test.ts

```diff
@@ -87,7 +87,7 @@ describe( `${ block.name } Block`, () => {
 			const isRefreshed = jest.fn( () => void 0 );
 			page.on( 'load', isRefreshed );
 			await setMaxPrice();
-			await page.waitForTimeout( 1000 );
+			await page.waitForNetworkIdle();
```

Comment:
should we move it inside of setMaxPrice() ?

Reply:
I'm not sure. For the PHP template, the setMaxPrice triggers a page refresh

Reply:
ah, alright :)